rcl_logging_spdlog/src/rcl_logging_spdlog.cpp

```diff
@@ -128,6 +176,17 @@ rcl_logging_ret_t rcl_logging_external_initialize(
 
     auto sink = std::make_unique<spdlog::sinks::basic_file_sink_mt>(name_buffer, false);
     g_root_logger = std::make_unique<spdlog::logger>("root", std::move(sink));
```

Comment:
`std::make_shared`?

Reply:
Yup, good catch. This should be safe as is but it would be better to use make shared.

Reply:
[49b8358](https://github.com/ros2/rcl_logging/pull/95/commits/49b83585f88f119acf8ce892f9771370680be22c)